tests/imputers/test_NearestMeanResponseImputer.py

```diff
@@ -1,14 +1,23 @@
+import copy
+
 import numpy as np
 import pandas as pd
 import pytest
 import test_aide as ta
 
 import tests.test_data as d
-import tubular
+from tests.base_tests import (
+    ColumnStrListInitTests,
+    GenericFitTests,
+    GenericTransformTests,
+)
+from tests.imputers.test_BaseImputer import (
+    GenericImputerTransformTests,
+)
 from tubular.imputers import NearestMeanResponseImputer
 
 
-# Dataframe used exclusively in this testing script
+# Dataframes used exclusively in this testing script
 def create_NearestMeanResponseImputer_test_df():
```

Comment:
Q: does this fill a different purpose to create_test_numeric_df_2?

Reply:
Mmm the answer is no! I removed that and also got rid of 'test_learnt_values2' as it was testing the same behaviour just with two different expected results 

[here](https://github.com/lvgig/tubular/pull/281/commits/1441ed28f1b074aa0314149b4306088e2787fca4)